os/binfmt/elf.c

```diff
@@ -295,6 +295,7 @@ static int elf_loadbinary(FAR struct binary_s *binp)
 	up_addrenv_clone(&loadinfo.addrenv, &binp->addrenv);
 #else
 	binp->alloc[ALLOC_TEXT] = (FAR void *)loadinfo.textalloc;
+	binp->textsize = loadinfo.textsize;
```

Comment:
Looks it should be under #if defined(CONFIG_SUPPORT_COMMON_BINARY) || defined(CONFIG_OPTIMIZE_APP_RELOAD_TIME). Please correct me if i'm wrong.

Reply:
Modified. Thanks.